crates/masonry/src/event_loop_runner.rs

```diff
@@ -49,6 +49,7 @@ pub fn run(
     let event_loop_proxy = event_loop.create_proxy();
     let adapter = Adapter::with_event_loop_proxy(&window, event_loop_proxy);
     window.set_visible(visible);
+    window.set_ime_allowed(true);
```

Comment:
Add a TODO to enable this on a more fine-grained basis - primarily when focusing on a text edit widget.